src/Command/ListMissingDownloadsCommand.php

```diff
@@ -104,14 +108,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         return 0;
     }
 
+    /**
+     * @return bool|string
+     */
     private function getFixedUrl(string $url, string $release, int $format)
```

Comment:
Doesn't the site run oh PHP 8 yet? :smile:

Reply:
No, I fear there are many packages not ready for PHP 8, the server is ready. Will be one of the next steps to update to PHP 8. Currently it's PHP 7.4.

Reply:
Changed to a single return type :)